class APIRequestError(Exception):
    """Вызывается при ошибке запроса к API Practicum."""
    pass